MediaAdminBundle/Transformer/FolderTransformer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminBundle\Transformer;
+
+use OpenOrchestra\ApiBundle\Exceptions\TransformerParameterTypeException;
+use OpenOrchestra\BaseApi\Transformer\AbstractTransformer;
+use OpenOrchestra\Media\Model\FolderInterface;
+use OpenOrchestra\MediaAdminBundle\Facade\FolderFacade;
+
+/**
+ * Class FolderTransformer
+ */
+class FolderTransformer extends AbstractTransformer
+{
+    const ROOT_FOLDERS_PARENT_ID = '-';
```

Comment:
i think is better to move this constant in FolderInterface and use it also in template twig